shared/src/bundles/user-details/types/user-details-update-request-dto.type.ts

```diff
@@ -1,49 +1,12 @@
 import { type ValueOf } from '~/types/value-of.type.js';
 
-import {
-    type CountryList,
-    type EmploymentType,
-    type EnglishLevel,
-    type JobTitle,
-    type NotConsidered,
-    type OnboardingSteps,
-    type PreferredLanguages,
-} from '../user-details.js';
+import { type OnboardingSteps } from '../user-details.js';
+import { type UserDetailsUpdateDto } from './types.js';
 
 type UserDetailsUpdateRequestDto = {
-    id?: string;
-    userId?: string;
-
-    isHired?: boolean;
-
-    profileName?: string;
-
-    salaryExpectation?: number;
-    hiredSalary?: number;
-
-    jobTitle?: ValueOf<typeof JobTitle>;
-    location?: ValueOf<typeof CountryList>;
-
-    experienceYears?: number;
-    employmentType?: ValueOf<typeof EmploymentType>[];
-
-    description?: string;
-
-    englishLevel?: ValueOf<typeof EnglishLevel>;
-    notConsidered?: ValueOf<typeof NotConsidered>[];
-    preferredLanguages?: ValueOf<typeof PreferredLanguages>[];
-
-    projectLinks?: string[];
-    photoId?: string;
-    fullName?: string;
-    phone?: string;
-    linkedinLink?: string;
-    companyName?: string;
-    companyLogoId?: string;
-    companyWebsite?: string;
-    employerPosition?: string;
-    cvId?: string;
+    talentBadges?: string[];
```

Comment:
I guess that's can be as below example or not?
```
    talentBadges?: {
        userId: string;
        score: number;
        level: string;
        badgeId: string;
    }[];
```

Reply:
as I understood @MaksymGavatuk about talent badges, we should recieve from frontend only array of strings = Ids

Reply:
Yes it just string[]